conf/pam_conv1/pam_conv_y.y

```diff
@@ -78,8 +78,10 @@ line
     /* $1 = service-name */
     yyerror("Appending to " PAM_D "/%s", $1);
 
-    filename = malloc(strlen($1) + sizeof(PAM_D) + 6);
-    sprintf(filename, PAM_D_FILE_FMT, $1);
+    if (asprintf(&filename, PAM_D_FILE_FMT, $1) < 0) {
+	yyerror("unable to create filename - aborting");
+	exit(1);
+    }
```

Comment:
This fix goes beyond the integer handling fixes.

Reply:
Moved into https://github.com/linux-pam/linux-pam/pull/671